docs/index.rst

```diff
@@ -131,31 +131,8 @@ as well as coding, documentation, and testing guidelines.
 
 {% if is_development %}
 
-For the guiding vision of this process and the project
-as a whole, see :doc:`development/vision`.
-
-.. toctree::
-   :maxdepth: 1
-
-   development/workflow/development_workflow
-   development/workflow/get_devel_version
-   development/when_to_rebase
-   development/codeguide
-   development/docguide
-   development/style-guide
-   development/testguide
-   testhelpers
-   development/scripts
-   development/building
-   development/ccython
-   development/releasing
-   development/workflow/maintainer_workflow
-   development/astropy-package-template
-   changelog
-
-There are some additional tools, mostly of use for maintainers, in the
-`astropy/astropy-procedures repository
-<https://github.com/astropy/astropy-procedures>`__.
+The developer/contributor documentation has been moved to its own page at
```

Comment:
While the previous list was definitely too long, I would love to keep the main links on this page; I use these frequently!

Reply:
From now on you would go to docs.astropy.org/development

Reply:
> From now on you would go to docs.astropy.org/en/latest/development

My point was that I frequently used the general docs.astropy.org page and could jump immediately to the page of interest; now I need another step. To me, the original design makes sense - have links to important pieces of each category; the only problem was that  small/unimportant pieces had accumulated.